src/battle_anim_electric.c

```diff
@@ -1452,3 +1482,40 @@ static void AnimShockWaveLightning(struct Sprite *sprite)
         DestroySprite(sprite);
     }
 }
+
```

Comment:
```suggestion

// [Explanation of what this task does here]
// Used by: Ion Deluge
// arg 0: 
// arg 1:
// arg 2:
```

Reply:
I meant that you should fill in this text.
What the `AnimTask` does and what the 3 arguments mean.